docker/server/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get -y install python-pip software-properties-common \
+ca-certificates apt-transport-https apt-utils \
+libterm-readline-gnu-perl software-properties-common \
+wget curl sudo
+
+RUN pip install urllib3 cryptography --upgrade
```

Comment:
pip3? (same in the other files as well)

Reply:
The base image is ubuntu 18.04 (same as the original Turbinia base), that does not contain Python3.

Reply:
Yep, but we should run this under python3. Fine to do later though. In TS I'm installing python3 in the Docker file:

```
    python3-dev \
    python3-pip \
    python3-wheel \
    python3-setuptools \
```

Reply:
Done